plugin-api/src/main/java/org/sonar/api/web/ServletFilter.java

```diff
@@ -23,6 +23,11 @@
 import org.sonar.api.server.ServerSide;
 
 /**
+ * Plugins can implement filters to serve HTTP requests that would otherwise not be served by the platform. This extension point
+ * is not intended to intercept all requests.
+ * When the platform intercepts and serves a request (for example with a web service filter), plugin filters do not get invoked,
+ * as the filters defined by plugins are at the end of the filter chain.
+ * Plugin filters get invoked when no other filters are interrupting the chain and returning a response to the request.
```

Comment:
I see that `org.sonar.api.web.ServletFilter` is deprecated and should be replaced by `org.sonar.api.web.HttpFilter`, and the javadoc you added is more complete than the new component, should we align both docs? Or they have different behaviors maybe?

Reply:
good point. I added it to `HttpFilter` too and I'm updating the jira ticket.